e2e/testcafe-devextreme/tests/dataGrid/common/accessibility/contrast.ts

```diff
@@ -54,3 +54,51 @@ fixture.disablePageReloads`DataGrid - contrast`
     },
   );
 });
+
+// T1286345
+[
+  Themes.genericLight,
+  Themes.fluentBlue,
+  Themes.materialBlue,
+].forEach((theme) => {
+  test('DataGrid - Filter icon should remain visible when it\'s focused', async (t) => {
+    const dataGrid = new DataGrid('#container');
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await t
+      .expect(dataGrid.isReady())
+      .ok();
+
+    // eslint-disable-next-line max-len
+    const searchIconContainer = dataGrid.getHeaders().getFilterRow().getFilterCell(1).getSearchIcon().element;
```

Comment:
Let's move method call on new line instead of disabling ESlint rule:

```suggestion
    const searchIconContainer = dataGrid
      .getHeaders()
      .getFilterRow()
      .getFilterCell(1)
      .getSearchIcon()
      .element;
```

Reply:
done ✔️